index.html

```diff
@@ -209,6 +209,21 @@ <h2>Basic Structure</h2>
     <h2>Digest Algorithm Choice</h2>
   </section>
 
+  <section id='digest-sidecar-file'>
+    <h2>Digest Sidecar File</h2>
+    <p>
+        Every occurrence of an inventory file MUST have an accompanying sidecar file stating its checksum. This sidecar file must be of the form <pre>inventory.jsonld.HASHALG</pre>, where <pre>HASHALG</pre> follows the chosen hashing
```

Comment:
Is there a canonical list of hash algorithms we can reference?

Reply:
See comments on #21 -- I think the answer is a bit tricky but we will need to define the list, hopefully by reference to one or more other specs. I would suggest we merge this and solve that as a separate problem